dbms/src/Flash/Planner/Plans/PhysicalJoin.cpp

```diff
@@ -311,6 +311,7 @@ void PhysicalJoin::buildPipeline(PipelineBuilder & builder, Context & context, P
     join_build_builder.build();
 
     // Join probe pipeline.
+    builder.setHasPipelineBreakerWaitTime(true);
```

Comment:
Also for aggregation?

Reply:
For aggregation, I think the wait time is not "real wait", since the aggregation calculation is actually always going.

Reply:
If so, then the real pipeline breaker is actually just join, and it feels like it can be directly named join-build-wait-time.
And then `hasPipelineBreakerWaitTime` is useless because only `source->type == Join` will add the wait-time

Reply:
Confirmed with SeaRise offline, we can't use source->type == Join to check. Since it only works for JoinProbe pipeline, will finally show to users like "join wait build time" in execution summary string.